appveyor.ps1

```diff
@@ -7,6 +7,33 @@ param(
 $global:ProgressPreference = 'SilentlyContinue'
 if($ShowProgress){$ProgressPreference = 'Continue'}
 
+function Get-ContentWithoutHeader {
+    param(
+      $path
+    )
+
+    $doc = Get-Content $path -Encoding UTF8
+    $start = $end = -1
+
+    for ($x = 0; $x -lt 30; $x++) {
```

Comment:
Can you add a comment why 30 was chosen?

Reply:
No Yaml header is 30 lines long so that should be plenty. Added comment to the code.